chromadb/test/property/test_filtering.py

```diff
@@ -305,3 +367,72 @@ def test_boolean_metadata(api: ServerAPI) -> None:
     res = coll.get(where={"test": True})
 
     assert res["ids"] == ["1", "3"]
+
+
+@settings(
+    suppress_health_check=[
+        HealthCheck.function_scoped_fixture,
+        HealthCheck.large_base_example,
+    ]
+)
+@given(
+    collection=collection_st_lf,
+    record_set=recordset_st_lf,
+    filters=st.lists(
+        strategies.filters(collection_st_lf, recordset_st_lf, include_all_ids=True),
+        min_size=1,
+    ),
+)
+def test_filterable_metadata_query_lf(
```

Comment:
Same as above. Are there any downsides of using the existing `test_filterable_metadata_query` ?